packages/fast-usdc/src/types.ts

```diff
@@ -104,6 +104,11 @@ export type FeeConfig = {
   variableRate: Ratio;
   /** proportion of fees that goes to the contract (remaining goes to LPs) */
   contractRate: Ratio;
```

Comment:
I suggest updating docs for `contractRate` to note that it's not the whole story w.r.t. contract fees; `relay` also goes to the contract.